### ISSUE TITLE
Add a 'where did you first hear of us' text area to sign up flow

### ISSUE THREAD

**Issue by jamesefhawkins** [PostHog/posthog#7873]
## Is your feature request related to a problem?

We don't have a great grasp of where our users _actually_ first hear about us.

As much as we can try to attribute through UTM tags, the reality is people will have read an article about us on HackerNews and then later googled us.

If we can understand this better, we can focus our spend on dev rel and marketing properly.

## Describe the solution you'd like

A mandatory text area in the sign up flow when the organization is created (for cloud) that asks where users first _heard_ of us. It should be a free form text field so we get more open answers.

## Describe alternatives you've considered

Spend marketing money in the wrong places, potentially.

**Comment by charlescook-ph** [comment]
This would be hugely valuable for tracking word of mouth, as asking for the qualitative type of info is our best way of figuring this stuff out (just scanning through this every week would really help the marketing team).

**Comment by clarkus** [comment]
I am all for answering this question, but I would encourage us to ask it after the main objectives of onboarding are complete. Anytime we add another field, it can increase the chances a user abandons the onboarding workflow. I think this kind of thing could work really well if we found thoughtful ways to ask in non-disruptive ways after a user has successfully signed up, ingested events, invited others, etc. 

When we ask for this info, I do think it should be optional. While a text field could encourage more open answers, it might also encourage a lot of "asdf" and meaningless entries just to dismiss the prompt. The flip side is a predefined list of options might also bias a user towards whatever values generally applies. Will think about ideas for this. 

Are there any other things we'd like to prompt users to answer? Company size, product area, etc? These are all things we could ask for after onboarding / sign up in a way that doesn't deter from completing the primary goals of onboarding.

**Comment by charlescook-ph** [comment]
Yep makes sense, I don't have a strong preference for when in onboarding this happens, just would be good to ask this in a non-obtrusive way (and presumably we'd want to check and make sure it doesn't unduly harm conversion). 

I don't think there are other questions at this point we need to answer that we can't already triangulate from sources like Clearbit/HubSpot. 

@kappa90 I think you were looking at onboarding right - curious what you think?
